migrator/init/user_for_api.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
```

Comment:
Changed permissions 100644 → 100755
Why?

Reply:
Didn't do that. Can't imagine why it happened

Reply:
Changed premission to 644